packages/ui/react-ui-board/src/Grid/Tile.tsx

```diff
@@ -2,36 +2,86 @@
 // Copyright 2025 DXOS.org
 //
 
-import React from 'react';
+import { draggable } from '@atlaskit/pragmatic-drag-and-drop/element/adapter';
+import React, { useEffect, useRef, useState } from 'react';
 
+import { invariant } from '@dxos/invariant';
 import { type ThemedClassName } from '@dxos/react-ui';
-import { mx } from '@dxos/react-ui-theme';
+import { Card } from '@dxos/react-ui-stack';
 
 import { useGridContext } from './Grid';
 import { getGridRect } from './geometry';
-import { type HasId, type TileLayout } from './types';
+import { type Position, type HasId, type TileLayout } from './types';
 
-// TODO(burdon): Contains surface like Kanban.
-// TODO(burdon): Drag handles only visible on long hover.
+type DragState = 'idle' | 'dragging';
 
 export type TileProps<T extends HasId = any> = ThemedClassName<{
   item: T;
   layout: TileLayout;
-  onClick?: () => void;
 }>;
 
-export const Tile = ({ classNames, item, layout: { x, y, width = 1, height = 1 }, onClick }: TileProps) => {
-  const { grid } = useGridContext('Tile');
+export const Tile = ({ classNames, item, layout }: TileProps) => {
+  const { grid, zoom, onSelect, onDelete, onMove } = useGridContext(Tile.displayName);
+
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [dragState, setDragState] = useState<DragState>('idle');
+  useEffect(() => {
+    const element = ref.current;
+    invariant(element);
+    return draggable({
+      element,
+      canDrag: () => !zoom,
+      onDragStart: () => {
+        setDragState('dragging');
+      },
+      onDrop: ({
+        location: {
+          current: { dropTargets },
+        },
+      }) => {
+        setDragState('idle');
+        const position = dropTargets[0]?.data.position as Position;
+        if (position) {
+          onMove?.(item.id, position);
+        }
+      },
+    });
+  }, [zoom]);
+
+  // TODO(burdon): Title accessor?
+  const title = item.id;
 
   return (
-    <div
-      className={mx('absolute flex flex-col bg-inputSurface border border-separator rounded-sm shadow', classNames)}
-      style={getGridRect(grid, { x, y, width, height })}
-      onClick={onClick}
+    <Card.Root
+      ref={ref}
+      // TODO(burdon): Root should have no padding by default (leave that to Content?)
```

Comment:
`Card.Root` applies padding in order to facilitate flush DnD drop zones while giving the illusion of consistent spacing between the cards.

You don’t _need_ to use `Card.Root` (unlike with many other components), you can skip it and just render `Card.Content`.